.github/workflows/vscode.yml

```
@@ -0,0 +1,68 @@
+env:
+  VERSION: 1.24.${{ github.run_number }}
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    
+    - name: Update Version
+      working-directory: 
```

actionlint (1.7.7)

string should not be empty

(syntax-check)